enforcer/README.md

```diff
@@ -254,20 +280,22 @@ used between the Enforcer and Controller.
     //
     // Apply a policy bundle to the enforcer.
     //
-    // Policies are specifically ordered.  Thus the addition, removal, or 
-    // modification of one or more policy requires a new policy bundle to 
-    // be applied to the enforcer. The enforcer is responsible for 
-    // determining which policies have been added or removed and 
+    // Policies are specifically ordered.  Thus the addition, removal, or
+    // modification of one or more policy requires a new policy bundle to
+    // be applied to the enforcer. The enforcer is responsible for
+    // determining which policies have been added or removed and
     // modifying its state or the state of its plugins as necessary.
-    // If no PolicyVersions change, and no policies are added 
+    // If no PolicyVersions change, and no policies are added
     // or removed, then nothing is done.
     //
+
+COMMENT: Not every system uses ordered policies.  No need to motivate policy bundles by ordering.
```

Comment:
That is true and it works as long as the domain of policies don't overlap.  If they do, you need some rules to cover which policy wins when they contradict each other.  Thus we specify a specific ordering in which policies are applied.